src/cli/commands/add.js

```diff
@@ -70,6 +108,7 @@ export class Add extends Install {
    */
 
   async init(): Promise<Array<string>> {
+    this.addedPatterns = [];
```

Comment:
Can this be removed and instead use the `patterns` below and passed to the `savedPackages` function? From what I can tell the `Install.init` returns the same thing that is pushed onto `this.addedPatterns`.